config/quality/pmd/pmd-ruleset.xml

```diff
@@ -20,8 +20,8 @@
     <rule ref="rulesets/java/naming.xml">
         <!--<exclude name="AbstractNaming" />-->
         <exclude name="LongVariable" />
-        <!--<exclude name="ShortMethodName" />-->
-        <!--<exclude name="ShortVariable" />-->
+        <exclude name="ShortMethodName" />
```

Comment:
I leave those two lines commented out because short naming should be an exception. Things named like `s` or `c` are not good for reading. Even in a couple of lines.